lib/bolt/pal.rb

```diff
@@ -409,7 +409,7 @@ def get_plan_info(plan_name)
                      end
           params[name] = { 'type' => type_str }
           params[name]['sensitive'] = param.type_expr.instance_of?(Puppet::Pops::Types::PSensitiveType)
-          params[name]['default_value'] = param.value
+          params[name]['default_value'] = param.value if param.value
```

Comment:
This should be `unless param.value.nil?` to accommodate the value `false`.

Reply:
D'oh. One of these days I'll remember that. >_>